docker/test/performance-comparison/compare.sh

```diff
@@ -77,23 +77,25 @@ function restart
     while killall clickhouse-server; do echo . ; sleep 1 ; done
     echo all killed
 
+    # Disable percpu arenas because they segfault when the process is bound to
+    # a particular NUMA node: https://github.com/jemalloc/jemalloc/pull/1939
```

Comment:
But it should not already, since #15035 is merged (it updates jemalloc submodule to include backported version of the patch from the referenced pull request in upstream jemalloc)

Reply:
Great, I'll remove it.